frontend/src/components/createitem.component.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import axios from "axios";
+import { Button, Container, Row, Col } from "react-bootstrap";
+import "./CreateItem.css";
+
+const CreateItem = () => {
+  async function createItem(inventory) {
+    let response;
+    try {
+      response = await axios.post("http://localhost:5000/inventory", inventory);
+    } catch (error) {
+      console.log(error);
+    }
+    response.then((result) => {
+      /*let item = result["data"]["item"];*/
+      if (result !== undefined && result.status === 201);
+      /*setInventories([...inventories, inventory]);*/
```

Comment:
Delete this since it's not needed

Reply:
done on local file. Will commit and push along with next batch of changes.

Reply:
Cool you should resolve conflicts also